Display Module/graph.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib.pyplot as plt
+import functions as f
+
+
+# Input
+def show(s, a, c, d):
```

Comment:
You should store function values for example in gui module, because we'll need to change them with our transformations in future. 
And why can't you attach your plot to your gui?